web/src/installers/versions.js

```diff
@@ -107,6 +107,7 @@ module.exports.InstallerVersions = {
   ],
   weave: [
     // cron-weave-update-265
+    "2.8.1-20230130",
```

Comment:
this looks wrong. did the automation insert this or was it manual? if automation we should fix

https://github.com/replicatedhq/kURL/actions/workflows/update-weave.yaml
https://github.com/replicatedhq/kURL/blob/main/addons/weave/template/generate.sh

Reply:
Why it shows wrong?
The automation made that. 
I just built the images:

- Built via the [build-images](https://github.com/replicatedhq/kURL/actions/workflows/build-image.yaml) workflow using path [addons/weave/2.8.1/build-images/weave-kube](https://github.com/replicatedhq/kURL/tree/main/addons/weave/2.8.1/build-images/weave-kube).

- Built via the [build-images](https://github.com/replicatedhq/kURL/actions/workflows/build-image.yaml) workflow using path [addons/weave/2.8.1/build-images/weave-npc](https://github.com/replicatedhq/kURL/tree/main/addons/weave/2.8.1/build-images/weave-npc)

- Built via the [build-images](https://github.com/replicatedhq/kURL/actions/workflows/build-image.yaml) workflow using path [addons/weave/2.8.1/build-images/weaveexec](https://github.com/replicatedhq/kURL/tree/main/addons/weave/2.8.1/build-images/weaveexec)

And then, re-executed the generate.sh script to create this PR to add the new version with the latest images built.

Reply:
I see the script added a duplicated entry. Good point.